task11/Client.ts

```diff
@@ -0,0 +1,19 @@
+import { Shipment } from './Shipment';
+import { DoNotLeaveShipmentDecorator } from './decorators/DoNotLeaveShipmentDecorator';
+import { FragileShipmentDecorator } from './decorators/FragileShipmentDecorator';
+import { ReturnReceiptShipmentDecorator } from './decorators/ReturnReceiptShipmentDecorator';
+
+
+let item: Shipment = new Shipment({
+    shipmentId: 17263,
+    fromAddress: '12292 4th Ave SE, Bellevue, Wa',
+    fromZipCode: '92021',
+    toAddress: '1313 Mockingbird Lane, Tulsa, Ok',
+    toZipCode: '67721',
+    weight: 170,
+});
+item = new FragileShipmentDecorator(item);
+item = new DoNotLeaveShipmentDecorator(item);
+item = new ReturnReceiptShipmentDecorator(item);
+item.ship(); 
```

Comment:
I suppose we could live you variant, but I suggest providing a similar format for all the marks which could be in the item.
If an array of marks is not empty, all marks should have the following format: `**MY SUPER MARK**`, not only special marks from decorators.
I suppose the same format would be better here.